snarkos/ledger/server.rs

```diff
@@ -196,6 +195,12 @@ impl<N: Network> Server<N> {
             .and(with(ledger_sender.clone()))
             .and_then(Self::transaction_broadcast);
 
+        // GET /testnet3/program/program_id
```

Comment:
Look at how the `get_transaction` endpoint for parses a transactionID, and apply that to the programID here.